README.md

```diff
@@ -125,10 +125,14 @@
 
 #### Casual
 
+<!--lint ignore double-link-->
 - KINGA - [Android](https://play.google.com/store/apps/details?id=com.knoeyes.games.kinga), [iOS](https://apps.apple.com/us/app/kinga/id1506422810) - Protect the cookie from the pesky flies. By [Kevin Omyonga](https://kevinomyonga.com)
 
 - Dino Run! - [Android](https://play.google.com/store/apps/details?id=io.github.epsi95.dinorun), - You are already familiar with this game, you can play it in Chrome when there is no internet. I just added a little color to it. Also you can shout to make the little cute Dino to jump, isn't that cool?. By [Probhakar Sarkar](https://github.com/epsi95)
 
+<!--lint ignore double-link-->
```

Comment:
Since we are listed on the official awesome list, I am not sure if we can ignore link rules, @impulse any idea on why this link rule broke here?

Reply:
@erickzanardo the linting tool introduced some bugs thus those lines were added. Maybe there is another way to work around this now.